__tests__/app.test.js

```diff
@@ -233,4 +233,50 @@ describe("POST /api/articles/:article_id/comments", () => {
             expect(res.body.msg).toBe('Article not found and comment cannot be posted')
         })
     })
+})
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("status 200: updates vote count when passed a positive value", () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ votes: 42})
+        .expect(200)
+        .then((res) => {
+            expect(res.body.article.votes).toBe(142)
+        })
+    })
```

Comment:
This is a good test but we still want to ensure that all of the properties we are expecting back on this endpoint so there should be an assertion for this here as well